modules/apps/roles/roles-admin-test/src/testIntegration/java/com/liferay/roles/admin/internal/exportimport/data/handler/test/RolesAdminPortletDataHandlerTest.java

```diff
@@ -97,10 +96,17 @@ protected void checkManifestSummaryReferrerClassNames(
 		for (String manifestSummaryKey :
 				manifestSummary.getManifestSummaryKeys()) {
 
-			Stream<String> classNamesStream = classNames.stream();
+			boolean hasMatch = false;
 
-			Assert.assertTrue(
-				classNamesStream.anyMatch(manifestSummaryKey::endsWith));
+			for (String className : classNames) {
+				if (manifestSummaryKey.endsWith(className)) {
+					hasMatch = true;
+
+					break;
+				}
+			}
+
+			Assert.assertTrue(hasMatch);
```

Comment:
@jeffwu0724 You do not need to have this variable "hasMatch", assert directly.

Reply:
Hi @tinatian ,

For this stream usage, we are using "anyMatch()". 
In my understanding, for all the **manifestSummaryKey**s, if we have one **className** that `manifestSummaryKey.endsWith(className)`, then it will be true. If we assert directly in the for loop, it is more like using "allMatch()" in this case. 
Please correct me if there is any error in my thought.
Thanks!

Reply:
@tinatian Here we might not be able to use assert directly like what Jeff did in https://github.com/liferay-core-infra/liferay-portal/pull/3269/commits/0a744ffd10dd9934be021ded843e7b424ab05aae.

`Assert.assertTrue(result)` throws `AssertionError` when the `result` is false. In this for-loop, if the first element in `classNames` doesn't make `manifestSummaryKey::endsWith` return ture, then `AssertionError` will be thrown, and the loop will exit -- even if any other element in the list does match the condition.

If we do want to avoid the `boolean hasMatch`, maybe we can do

```
List<String> filteredClassNames = ListUtil.filter(classNames, manifestSummaryKey::endsWith);

Assert.assertTrue(!filteredClassNames.isEmpty());
```

cc @jeffwu0724